spec/support/vcr_cassettes/travel_pay/200_claims.yml

```diff
@@ -179,7 +179,7 @@ http_interactions:
             "modifiedOn": "2024-04-23T16:44:34.465Z"
           },
```

Comment:
Can you please replace the URI with an variable that can be defined in the [vcr spec](https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/support/vcr.rb#L3).

Reply:
Can I clarify? You want us to:

* Add this URL to the vcr spec: https://auth.veis.gov/tenant_id/oauth2/token
* Replace all instances of the that url with the variable in this file

Is that right @RachalCassity?

Reply:
@kjduensing Yes, please. At least replace the domain, https://auth.veis.gov/.

Reply:
Thanks for clarifying, Rachal! 

cc @athifw